### ISSUE TITLE
Bug playing RTBF videos on release 0.0.32

### ISSUE THREAD

**Issue by ze-big-ant** [zyphos/belgium-replay-tv#26]
Hi,
you fixed a similar bug yesterday with release 0.0.32 but there was a new problem now:

[kodi.log](https://github.com/zyphos/belgium-replay-tv/files/3009237/kodi.log)

I fixed that by creating folder /home/osmc/.kodi/userdata/addon_data/plugin.video.belgium/ and file history.dump. Now, There's another bug:

[kodi.log](https://github.com/zyphos/belgium-replay-tv/files/3009321/kodi.log)

Thanks for your help.

**Comment by zyphos** [comment]
I checked this version without bug against:
- Ubuntu 16.04 on Intel platform, Kodi 18.0
- Android 7.1.2, Kodi 18.0
- Libreelec 9.0.1 on Raspberry pi 2, Kodi 18.0

This error is due to the file history.dump created empty, this can only be done by tweaking yourself.
But I can add more safe check to avoid it, like disabling history if folder is not writable, but this situation can not exists on normal installation.

To fix this error, remove the history.dump file, and check that kodi has write access to folder /home/osmc/.kodi/userdata/addon_data/plugin.video.belgium/

**Comment by ze-big-ant** [comment]
Yes, it works just fine. Thanks a lot for your help and for your plugin!
